medusa/common.py

```diff
@@ -438,6 +438,7 @@ def quality_from_file_meta(file_path):
         bluray = re.search(r"blue?-?ray|hddvd|b[rd](rip|mux)", base_filename, re.I) is not None
         webdl = re.search(r"web.?dl|web(rip|mux|hd)", base_filename, re.I) is not None
 
+        height = height.magnitude
```

Comment:
This should probably moved up to just after `if height:` and should probably be type-cast to `int`.  Also for knowit, I would recommend a naive version that doesn't require the  pint library or an option to disable it.